src/client/sky.cpp

```diff
@@ -717,14 +702,106 @@ void Sky::draw_moon(video::IVideoDriver *driver, float moonsize, const video::SC
 	}
 }
 
+void Sky::draw_stars(video::IVideoDriver * driver, float wicked_time_of_day)
+{
+	driver->setMaterial(m_materials[1]);
+	// Tune values so that stars first appear just after the sun
+	// disappears over the horizon, and disappear just before the sun
+	// appears over the horizon.
+	// Also tune so that stars are at full brightness from time 20000
+	// to time 4000.
+
+	float tod = wicked_time_of_day < 0.5f ? wicked_time_of_day : (1.0f - wicked_time_of_day);
+	float starbrightness = clamp((0.25f - fabs(tod)) * 20.0f, 0.0f, 1.0f);
```

Comment:
```
[...]/src/client/sky.cpp:715:25: error: no matching function for call to 'clamp' [clang-diagnostic-error]

        float starbrightness = clamp((0.25f - fabs(tod)) * 20.0f, 0.0f, 1.0f);
```
Worth fixing clang. https://travis-ci.org/minetest/minetest/jobs/654161899

Reply:
`fabs` -> `fabsf` should do the trick

I can also do this on merge